src/main/java/org/junitpioneer/jupiter/TempDirectory.java

```diff
@@ -125,7 +125,8 @@
 		 *
 		 * @return the parent directory for all temporary directories
 		 */
-		Path get(ParameterContext parameterContext, ExtensionContext extensionContext) throws Exception;
+		Path get(ParameterContext parameterContext, ExtensionContext extensionContext) throws Exception; // NOSNAR
```

Comment:
```suggestion
		Path get(ParameterContext parameterContext, ExtensionContext extensionContext) throws Exception; // NOSONAR
```

Reply:
Ops yes. And I think my other `// NOSONAR` where wrong too, but not sure. Anyway changed them to `//NOSONAR`